docs/plans/add_labels_to_intensity_timeline/walkthrough.md

```
@@ -0,0 +1,25 @@
+### [IntensityTimeline コンポーネント]
+
+#### [MODIFY] [IntensityTimeline.tsx](file:///home/lindq/develop/sake-sense/app/src/components/IntensityTimeline.tsx)
+
+- グラフの左側に縦軸ラベル「強」「弱」を追加。
+- グラフの下側に横軸ラベル「含んだ瞬間」「余韻」を追加。
+- Flexbox（`flex gap-2`）を使用して、ラベルとグラフを整列させ、適切な余白（`mt-2` など）を設定しました。
+- スタイル:
+    - フォントサイズ: `10px`
+    - 色: `slate-400`
+    - ユーザーによる選択を防止するための `select-none` を適用。
+
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)